specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json

```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```

Comment:
Looks like a BOM got added

Reply:
Will fix. My mistake.

Reply:
NP, you might need to split this up to 1 per spec, because I think it doesn't process when you have multiple APIs changed in the same PR

Reply:
Split the commits as requested. Thank you for your patience.

Reply:
Oh, sorry, I mean they might need to be in separate PRs for those messages "This PR contains more than 3 context, SDK generation is not enabled. Contexts found:"

Reply:
Ah, I misread. My apologies about this. I will close this PR for now, and re-create the individual PRs later today after I finish some other work.